provider/ec2/environ.go

```diff
@@ -1804,7 +1804,7 @@ func (e *environ) IngressRules(ctx context.ProviderCallContext) (firewall.Ingres
 }
 
 func (*environ) Provider() environs.EnvironProvider {
-	return &providerInstance
+	return &environProvider{}
```

Comment:
Why?

Reply:
I probably was trying to test something. I fixed this

Reply:
The change was accidental and was not intended to be pushed. Sorry for the confusion.